### DISCUSSION TITLE
Is the cracked file just the original steam file?

### DISCUSSION THREAD

**Discussion by ghost** [BigBoiCJ/SteamAutoCracker#11] [Q&A] [answered]
Hope you dont mind a question from a complete noob. SteamAutoCrack works just fine like the example you showed, but im slightly confused as to where the cracked file is. Am I safe to assume its just the steam folder the game was originally in? Can I just copy that file to another drivecomputer and play it drm free without it going through steam?

**Comment by BigBoiCJ** [comment, marked as answer]
Yes, you can move a game to a different computer and the crack will still work.

SAC modifies 2 things:
- The `steam_api.dll` and `steam_api64.dll` files located inside the game folder (and adds a crack config file next to those). **When cracking a game, SAC should write `- Found Steam API DLL(s) in 'location'`. Check that location ;)**
- The game's executable file (not all games) (also located inside the game folder).

SAC doesn't do anything more than modify some files inside a game folder, so yes, you can easily copy that game folder to another drive or computer, and the crack will still work.

However, I do not recommend trying to copy the crack file from one game to another, as you'll have to also configure the crack config. It's better to let SAC handle that part (unless you know what you're doing).

Be aware though that Steam updates can remove the crack, so you should consider moving the game folder somewhere Steam isn't installed.

---

Asking is perfectly fine, whatever your experience is. I'm not very experimented with cracking and all that myself, I just have some basic knowledge that made me able to do this tool.
